Tests/PINAnimatedImageTests.swift

```diff
@@ -189,4 +189,15 @@ class PINAnimatedImageTests: XCTestCase, PINRemoteImageManagerAlternateRepresent
         let webpAnimatedImage = PINWebPAnimatedImage(animatedImageData: data)
         XCTAssert(webpAnimatedImage == nil)
     }
+
+    func test_retainCycle() {
+        weak var weakCachedAnimatedImage : PINCachedAnimatedImage?
+        autoreleasepool {
+            let animatedImage = TestAnimatedImage.init()
+            let cachedAnimatedImage = PINCachedAnimatedImage.init(animatedImage: animatedImage)
+            weakCachedAnimatedImage = cachedAnimatedImage
+        }
+        let cachedAnimatedImage : PINCachedAnimatedImage? = weakCachedAnimatedImage
```

Comment:
I had to do this since it seems maybe swift does not `nil` out the weak reference until needed.